tools/cellranger.wdl

```diff
@@ -20,35 +18,43 @@ task count {
     }
 
     input {
-        String id
-        File transcriptome_tar_gz
         File fastqs_tar_gz
-        Int ncpu = 1
+        File transcriptome_tar_gz
+        String id
+        String jobmode = "local"  # TODO would a non-local jobmode work?
```

Comment:
I am fairly certain this parameter _must_ be set to `local`. In which case we should hard code instead of exposing it like this. Or am I wrong and other viable `jobmode` options exist?